content/04.model-complexity.md

```diff
@@ -0,0 +1,35 @@
+### Techniques and procedures must be implemented to manage model complexity without sacrificing the value of machine learning
+Inherent challenges posed by low sample numbers in rare diseases are further aggravated by disease heterogeneity, poorly defined disease phenotypes, and often a lack of control (i.e. normal) data. 
+Machine learning approaches must be carefully designed to address these challenges. 
+We discuss how to implement methodological solutions like bootstrapping sample data, regularization methods for deep learning, and hyper-ensemble techniques to minimize misinterpretation of the data. 
+
+#### Bootstrapping
+
+Bootstrap or resampling computation is a powerful statistical technique that can be used for estimating population values from datasets of limited sample size by resampling the data to generate a distribution of the population statistic and minimize estimation error [@doi:10.1080/01621459.1997.10474007].
+Bootstrap based techniques are used in conjunction with various learning methods to find the most informative models given a specific dataset (e.g. bootstrap aggregating or bagging used in random forests [@doi:10.1023/A:1010933404324; @doi:10.1198/0003130043277], bootstrap in neural networks [@doi:10/c8xpqz], or regression models [@doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
+In addition to model selection, rare disease datasets also necessitate the use of bootstrap to form an informative dataset [@doi:10.3390/genes11020226].
+In this study, bootstrapping the training sample without replacement simulated separate incomplete datasets that helped expose the learning models (in this case random forests) to the incompleteness of the data.
+Thus additional bootstrapping of the training data helped create confidence intervals for the predictions and the important predictors originating from the ensemble models run on the incomplete training data.
+
+#### Regularization
+
+Incompleteness of data in rare disease has led to efforts for collating data points dispersed over sites or time points to produce a statistically powerful dataset. 
+Given a limited dataset with strong pre-existing site-specific technical differences between groups of samples, machine learning models may optimize learning technical differences instead of real patterns leading to high prediction accuracy for training data (also termed low bias in the model or "overfit") [@doi:10.1073/pnas.1900654116].
```

Comment:
```suggestion
Given a dataset with strong preexisting study-specific technical differences between groups of samples, machine learning methods may model dataset-specific features instead of true biology, leading to high prediction accuracy for training data but poor performance in new test data (an "overfit" model) [@doi:10.1073/pnas.1900654116].
```